Rakefile

```diff
@@ -1,2 +1,7 @@
 require "bundler/gem_tasks"
 
+begin
+  require 'rspec/core/rake_task'
+    RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
+end
```

Comment:
But does it hurt having the rake task in there for consistency with using rake to build and release the gem?